spec/factories/course_lesson_plan_items.rb

```diff
@@ -8,6 +8,7 @@
     bonus_end_at { 1.day.from_now }
     end_at { nil }
     sequence(:title) { |n| "Example Lesson Plan Item #{n}" }
+    published true
```

Comment:
Actually I think quite a few specs relies on published being false by default. Suggest we add this as a trait instead?

Reply:
Sure.
Will setting the default to false, then overwriting it in that one spec work?

Reply:
Yup yup 👍